ParabolInc/parabol
# feat(meeting-inception): implement add an activity button

Fixes #8750

<img width="376" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/171b171b-d307-4abf-8f5f-83bb6c3fed78">


How to test:

- Add `meetingInception` org level feature flag
- Open various discussion panels, click on `add an activity` see it redirect to the activity library
